pkg/didcomm/crypto/jwe/authcrypt/decrypt.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package authcrypt
+
+import (
+	"golang.org/x/crypto/blake2b"
+	"golang.org/x/crypto/chacha20poly1305"
+	"golang.org/x/crypto/nacl/box"
+)
+
+// Decrypt will JWE decode the envelop argument for the sender and recipients
+// Using (X)Chacha20 encryption algorithm and Poly1035 authenticator
+func (c *Crypter) Decrypt(envelope []byte) (string, error) {
+	decryptOID(nil, nil, nil)
+	return "", nil
+}
+
+// decryptOID will decrypt a recipient's encrypted OID (in the case of this package, it is represented as
+// ephemeral key concatenated with the sender's public key) using the recipient's privKey/pubKey keypair,
+// this is equivalent to libsodium's C function: crypto_box_seal()
+// https://libsodium.gitbook.io/doc/public-key_cryptography/sealed_boxes#usage
+func decryptOID(privKey, pubKey *[chacha20poly1305.KeySize]byte, encrypted []byte) []byte {
+	var epk [32]byte
+	var nonce [24]byte
+	copy(epk[:], encrypted[:chacha20poly1305.KeySize])
+
+	nonceWriter, _ := blake2b.New(24, nil)
```

Comment:
error is being ignored here.

I have seen this in several places in the PR (mostly without comments).

Reply:
will add error handling..

Reply:
done